ibm/service/cos/resource_ibm_cos_bucket.go

```diff
@@ -185,25 +186,30 @@ func ResourceIBMCOSBucket() *schema.Resource {
 				Type:        schema.TypeList,
 				Optional:    true,
 				MaxItems:    1,
-				Description: "Enables sending log data to Activity Tracker and LogDNA to provide visibility into object read and write events",
+				Description: "Enables sending log data to IBM Cloud Activity Tracker to provide visibility into bucket management, object read and write events.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"read_data_events": {
 							Type:        schema.TypeBool,
 							Optional:    true,
 							Default:     false,
-							Description: "If set to true, all object read events will be sent to Activity Tracker.",
+							Description: "If set to `true`, all object read events (i.e. downloads) will be sent to Activity Tracker.",
 						},
 						"write_data_events": {
 							Type:        schema.TypeBool,
 							Optional:    true,
 							Default:     false,
-							Description: "If set to true, all object write events will be sent to Activity Tracker.",
+							Description: "If set to `true`, all object write events (i.e. uploads) will be sent to Activity Tracker.",
+						},
+						"management_events": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Description: "If set to `true`, all bucket management events will be sent to Activity Tracker.This field only applies if `activity_tracker_crn` is not populated.",
```

Comment:
can we add conflicts with here for `activity_tracker_crn` something like activity_tracking.0.activity_tracking
similar below one ` activity_tracking.0.management_events`

Reply:
@hkantare we cannot add a conflict because activity_tracker_crn provided and activity_tracking.0.management_events is true is a valid scenario , in this case the crn will take precedence. But usage of activity_tracker_crn is legacy behaviour which shouldn't be used. Hence instead of that management_events is the preferred way that is why activity_tracker_crn not being populated is the recommended way of adding activity tracker on a bucket.